test/verify/check-superuser

```diff
@@ -106,14 +107,14 @@ class TestSuperuser(MachineCase):
 
         # Configure the sudo PAM stack to make two prompts
         if "debian" in m.image or "ubuntu" in m.image:
-            m.write("/etc/pam.d/sudo", """
+            self.write_file("/etc/pam.d/sudo", """
```

Comment:
Ah, so the next commit addresses the remaining test :grin: This could just be squashed into the first commit.

Reply:
And please move the `@nd` to the class, instead of on each individual test.